### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by tron19920125** [ysmiura/ifcc#1]
Thanks for your great work. When I am following the tutorial to reproduce the results, some problems are raised. I would be appreciated if you tell me how to fix this issue.

Step:
`python train.py --cuda --corpus mimic-cxr --cache-data cache --epochs 32 --batch-size 24 --entity-match mimic-cxr_ner.txt.gz --img-model densenet --img-pretrained chexpert_densenet/model_auc14.dict.gz --bert-score distilbert-base-uncased --corpus mimic-cxr --lr-scheduler trans MIMIC_CXR_ROOT resources/glove_mimic-cxr_train.512.txt.gz out_m2trans_nll`

Return:
Segmentation fault (core dumped)

**Comment by ysmiura** [comment]
I am not exactly sure what has caused the segmentation fault but one possibility is that you may need to update packages relating to CUDA/cuDNN (pytorch, cudatoolkit, and torchvision) so that they are compatible with your runtime environment.
